server/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -917,6 +917,7 @@ public StoreStats storeStats() {
         try {
             return store.stats();
         } catch (IOException e) {
+            failShard("Failing shard because of exception during storeState " + e.getMessage(), e);
```

Comment:
This should say `storeStats`, not `storeState`.

Reply:
One more thing, looking at the implementation of `failShard`, we don't need to append `e.getMessage()` here.